scripts/generate_readme

```diff
@@ -27,12 +27,11 @@ if [ "$#" -ne 1 ]; then
   exit 1
 fi
 
-JAZZY_VERSION=`jazzy --version`
-if [[ $? != 0 ]]; then
-  echo "Cannot find jazzy. jazzy is required to generate API links. To install try:"
-  echo "[sudo] gem install jazzy"
+command -v jazzy >/dev/null 2>&1 || { 
```

Comment:
Was JAZZY_VERSION unused?

Reply:
Yes. The original line ("JAZZY_VERSION=`jazzy --version`") produced this error:
"line 31: jazzy: command not found"
instead of showing the "cannot find jazzy..." help message.

Reply:
Yes. The original line ("JAZZY_VERSION=jazzy --version") produced this error:
"line 31: jazzy: command not found"
instead of showing the "cannot find jazzy..." help message.